spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/AddFilesProcedure.java

```diff
@@ -165,12 +168,16 @@ private void importFileTable(
       boolean checkDuplicateFiles,
       PartitionSpec spec) {
     // List Partitions via Spark InMemory file search interface
+    long start = System.currentTimeMillis();
     List<SparkPartition> partitions =
         Spark3Util.getPartitions(spark(), tableLocation, format, partitionFilter, spec);
+    LOG.info("found {} partitions in {}", partitions.size(), Spark3Util.duration(start));
```

Comment:
I think such instrumentation is helpful when comparing performance, but do you think it's needed for all imports?

Reply:
yeah, meant to remove them. removed